### ISSUE TITLE
TypeError: unexpected kwargs: {'ssl': False}

### ISSUE THREAD

**Issue by ginkel** [ownaginatious/amqp-client-cli#4]
## Problem description

Attempting to send a message using `amqpcli` yields a `TypeError: unexpected kwargs: {'ssl': False}`:

```
$ amqpcli send localhost 5672 amq.direct queue -u guest -m "Test"
Connecting to queue @ localhost:5672... Traceback (most recent call last):
  File "/home/tg/.local/bin/amqpcli", line 33, in <module>
    sys.exit(load_entry_point('amqp-client-cli==0.3', 'console_scripts', 'amqpcli')())
  File "/home/tg/.local/lib/python3.9/site-packages/amqp_client_cli/main.py", line 179, in main
    app.run()
  File "/home/tg/.local/lib/python3.9/site-packages/clip.py", line 652, in run
    self.invoke(self.parse(tokens))
  File "/home/tg/.local/lib/python3.9/site-packages/clip.py", line 634, in invoke
    self._main.invoke(parsed)
  File "/home/tg/.local/lib/python3.9/site-packages/clip.py", line 523, in invoke
    self._subcommands[k].invoke(v)
  File "/home/tg/.local/lib/python3.9/site-packages/clip.py", line 519, in invoke
    self._callback(**{k: v for k, v in iteritems(parsed) if k not in self._subcommands})
  File "/home/tg/.local/lib/python3.9/site-packages/amqp_client_cli/main.py", line 112, in amqp_send
    connection = pika.BlockingConnection(pika.ConnectionParameters(
  File "/home/tg/.local/lib/python3.9/site-packages/pika/connection.py", line 658, in __init__
    raise TypeError('unexpected kwargs: %r' % (kwargs,))
TypeError: unexpected kwargs: {'ssl': False}
```

## Environment

```
$ uname -a
Linux vega 5.9.14-arch1-1 #1 SMP PREEMPT Sat, 12 Dec 2020 14:37:12 +0000 x86_64 GNU/Linux

$ python --version
Python 3.9.1
```

Thanks,
Thilo

**Comment by leferrad** [comment]
Same problem here

**Comment by ownaginatious** [comment]
Thanks for the heads up! Should be fixed in the latest version, `0.4.1`.
